helm/templates/configmap.yaml

```diff
@@ -7,12 +7,30 @@ metadata:
 data:
   application.conf: |-
     document.store {
-      dataStoreType = mongo
+      dataStoreType = {{ .Values.configMap.dataStoreType }}
+      {{- if eq .Values.configMap.dataStoreType "mongo" }}
       mongo {
         {{- if .Values.configMap.mongodb.url }}
         url = {{ .Values.configMap.mongodb.url | quote }}
         {{- else }}
         host = {{ .Values.configMap.mongodb.host }}
         {{- end }}
       }
+      {{ else if eq .Values.configMap.dataStoreType "postgres" }}
```

Comment:
Can this be generalized to just pick the key/value pairs under the specific dataStoreType and populate the entire section

Reply:
Done